plat/mediatek/mt6795/plat_delay_timer.c

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
This can be removed and drivers/delay_timer/generic_delay_timer.c used instead. See https://github.com/ARM-software/arm-trusted-firmware/pull/625/commits/1d0b990e9d7e28fdf0de588815cd21d88c9fc2b7